routes/email.js

```diff
@@ -6,12 +9,14 @@ process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 var transporter = nodemailer.createTransport({
   service: 'Gmail',
   auth: {
-    user: process.env.OBAY_GMAIL_USER || '',
-    pass: process.env.OBAY_GMAIL_PASS || ''
+    user: process.env.OBAY_GMAIL_USER,
+    pass: process.env.OBAY_GMAIL_PASS,
```

Comment:
These don't need to be removed. The OR statement would not reach this place if the process environment exists.